package.json

```diff
@@ -91,14 +91,14 @@
   "scripts": {
     "benchmark-rule": "node scripts/benchmark-rule.js",
     "flow": "flow",
+    "jest": "jest",
     "lint:js": "eslint .",
     "lint:md": "remark . --quiet --frail",
     "lint": "npm-run-all --parallel lint:*",
     "pretest": "npm-run-all --serial lint flow",
     "release": "npmpub",
-    "test:jest": "jest",
-    "test": "npm-run-all --serial test:*",
-    "watch": "npm run test:jest -- --watch"
+    "test": "jest",
+    "watch": "npm run jest -- --watch"
```

Comment:
Running `npm run watch` works as expected using `"watch": "npm run jest -- --watch"`

Running `npm run watch` does **not** work using `"watch": "npm run jest --watch"`, this results in the Jest tests being ran and the _watch_ mode is never started:

```shell
> stylelint@7.6.0 watch /Users/netweb/dev/stylelint/stylelint
> npm run jest --watch


> stylelint@7.6.0 jest /Users/netweb/dev/stylelint/stylelint
> jest

 PASS  lib/rules/value-keyword-case/__tests__/index.js
...
...
```